index.html

```diff
@@ -0,0 +1,243 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/main.css" />
+  <title>Aziz Mejri - Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <h1 id="logo">Aziz Mejri</h1>
+    <button type="button" class="menu-bars">
+      <i class="fas fa-bars"></i>
+    </button>
+  </header>
+  <main>
+    <div class="intro">
```

Comment:
This div could be a section tag instead as it makes it semantically more convenient. Kindly fix this.

Reply:
Done